packages/expo-av/android/src/main/java/expo/modules/av/player/datasource/DataSourceFactoryProvider.java

```diff
@@ -3,11 +3,12 @@
 import android.content.Context;
 
 import com.google.android.exoplayer2.upstream.DataSource;
+import com.google.android.exoplayer2.upstream.TransferListener;
 
 import java.util.Map;
 
 import org.unimodules.core.ModuleRegistry;
 
 public interface DataSourceFactoryProvider {
-  DataSource.Factory createFactory(Context reactApplicationContext, ModuleRegistry moduleRegistry, String userAgent, Map<String, Object> requestHeaders);
+  DataSource.Factory createFactory(Context reactApplicationContext, ModuleRegistry moduleRegistry, String userAgent, Map<String, Object> requestHeaders, TransferListener bandwidthMeter);
```

Comment:
I can find two implementations of this interface and only one is changes within this PR. I am afraid Expo Client won't be able to build :(

Reply:
I think I've found the other impl' and added the expected arg

Reply:
There is another one in `versioned.host.exp.exponent.modules.universal.av` package in `:expoview` module.

Reply:
https://github.com/expo/expo/pull/8363/files#diff-72aabec1f0463289beaa60675f0e93c3 the one here?

Reply:
sorry found it